linked-list/linked-list.spec.js

```diff
@@ -0,0 +1,72 @@
+import LinkedList from './linked-list';
+
+describe('LinkedList', () => {
+  it('push/pop', () => {
+    const list = new LinkedList();
+    list.push(10);
+    list.push(20);
+    expect(list.pop()).toBe(20);
+    expect(list.pop()).toBe(10);
+  });
+  xit('push/shift', () => {
```

Comment:
Could a better description be: `extract elements from the beginning of the list with shift`?